cmd/ddev/cmd/utils.go

```diff
@@ -0,0 +1,43 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/drud/ddev/pkg/ddevapp"
+	"github.com/drud/ddev/pkg/util"
+)
+
+func getRequestedApps(args []string, allFlag bool) ([]*ddevapp.DdevApp, error) {
+	// If allFlag is true, all active apps will be returned.
+	if allFlag {
+		if len(args) > 0 {
```

Comment:
I'm always amazed at how complicated cobra is. Seems like one shouldn't have to do this, that it should be in the cobra initialization. NO ARGS THANK YOU VERY MUCH.

Reply:
Oh, you're looking for args on the command itself. Not sure why `ddev rm proj1 proj2 --all` shouldn't work. But OK with this approach.